glean-core/src/ping/mod.rs

```diff
@@ -282,6 +283,10 @@ impl PingMaker {
             file.write_all(url_path.as_bytes())?;
             file.write_all(b"\n")?;
             file.write_all(::serde_json::to_string(ping_content)?.as_bytes())?;
+            if let Some(tag) = debug_view_tag {
+                file.write_all(b"\n")?;
```

Comment:
Can we split this out to a separate PR? I'm not sure it makes sense to handle this here and now: since we're going to add other tags that we'll need to persist, we might need to provide a flexible solution here from day 0, to not need to care about migrating pings in a different format.

Reply:
I removed this changes from this PR, we can discuss how to better persist this header value when I deal with that specific bug, as suggested. 

Now, this made python stop working, which makes sense since the tag is now on the Glean object and python actually instantiates a standalone instance of the Upload Manager for doing the uploads. I will have to drop the python changes from this PR too, so that nothing is broken. When we persist the headers then we can propagate this changes there.